cookbooks/tomography_based_plate_motions/2D_slice_with_faults_slabs_and_topo.prm

```diff
@@ -0,0 +1,168 @@
+# This cookbook demonstrates how we can set up instantaneous mantle flow models
+# using available geophysical constraints. To avoid large computational cost,
+# this model is a 2D spherical shell with imposed plate boundaries and driven
+# by mantle forces arising from observed mantle heterogeneity.
+# The generated mantle flow field at the surface can be compared with the observed
+# surface deformation and the fit can be used to constrain the physical state
+# of the mantle.
+# We define plate boundaries and cratons using WorldBuilder.
+
+include 2D_slice_with_faults_and_cratons.prm
+
+set Additional shared libraries            = ./plugins/libtomography_based_plate_motions.so
+set Dimension                              = 2
+set Use years in output instead of seconds = true
+set Output directory                       = 2D-slice-with-faults-slabs-topo
+set World builder file                     = $ASPECT_SOURCE_DIR/cookbooks/tomography_based_plate_motions/input_data/world_builder_smac_cratons_faults_2D.json
+set Nonlinear solver scheme                = iterated Advection and Stokes
+set Start time                             = 0
+set End time                               = 0
+set Max nonlinear iterations               = 30
+set Adiabatic surface temperature          = 1573.0
+
+# We use matrix-free solver and geometric multigrid preconditioner
+# to reduce memory consumption.
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Linear solver tolerance                         = 1e-4
```

Comment:
This value is on the relative low end, depending on the viscosity range and type of rheology. Can you add a comment on why it was decreased from the default value and something to the effect of "testing revealed that this value is sufficiently strict", etc?